examples/chefapi_examples/Berksfile

```diff
@@ -1,4 +1,3 @@
 source 'https://supermarket.chef.io'
-source :chef_server
```

Comment:
The only cookbooks ran on the Test Kitchen instance are in the `chefapi_examples` cookbook, so this source is unecessary.